### PR TITLE
Link mobility backend

### PR DESCRIPTION
**Pull request by toothfairy**

### CODE REVIEW

File: lib/cellular/backends.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@ module Backends
     autoload :CoolSMS, 'cellular/backends/cool_sms'
     autoload :Sendega, 'cellular/backends/sendega'
     autoload :Twilio,  'cellular/backends/twilio'
+    autoload :LinkMobility,  'cellular/backends/link_mobility'
     autoload :Log,     'cellular/backends/log'
     autoload :Test,    'cellular/backends/test'
```
Comment (jgorset) [inline lib/cellular/backends.rb]: We should indent all these to match line 8.

Comment (toothfairy) [inline lib/cellular/backends.rb]: It looks super ugly when it's indented :|

Comment (toothfairy) [inline lib/cellular/backends.rb]: Removed indentation for all the lines

**Comment by sindrenm** [comment]
This is so beautiful, btw. :heart:

**Pull request merged**